components/Map.tsx

```diff
@@ -0,0 +1,39 @@
+import { MapContainer, Marker, Popup, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import L from "leaflet";
+import { ArtistInterface } from "../lib/ArtistClass";
+import Link from "next/link";
+
+type MapProps = {
+  artists: ArtistInterface[];
+};
+
+export default function Map({ artists }: MapProps) {
+  var myIcon = L.icon({
```

Comment:
wir sollen kein var verwenden, oder? ist das in Typescript anders?

Reply:
Ich sags wie's ist: Copy and Paste an dieser Stelle! Werde das ändern, danke!